lib/ret/media_resolver.ex

```diff
@@ -204,13 +204,17 @@ defmodule Ret.MediaResolver do
       %HTTPoison.Response{headers: headers} ->
         content_type = headers |> content_type_from_headers
 
-        if !is_local_url && photomnemonic_endpoint && content_type |> String.starts_with?("text/html") do
-          case uri |> screenshot_commit_for_uri(content_type) do
-            :error -> uri |> og_tag_commit_for_uri()
-            commit -> commit
+        if content_type |> String.starts_with?("text/html") do
+          if !is_local_url && photomnemonic_endpoint do
+            case uri |> screenshot_commit_for_uri(content_type) do
+              :error -> uri |> og_tag_commit_for_uri()
+              commit -> commit
+            end
+          else
+            uri |> og_tag_commit_for_uri()
           end
         else
-          uri |> og_tag_commit_for_uri()
+          {:commit, uri |> resolved(%{expected_content_type: content_type})}
         end
```

Comment:
this changes the behavior for local URLs -- previously, if you had the lambda setup, it would not crawl a local URL and just return it. but if you didn't have the lambda setup (as is common in the local dev case) then it would crawl the URL, which was wrong.

Reply:
Hmm, doesn't the change to the outer else condition affect non-local URLs as well? Previously we would return the og_tag_commit if the content_type was not text/html, but now we return the resolved URI.

Reply:
ah that also seems like a bug, because we shouldn't be crawling pages that aren't text/html for OG tags (i don't think so, at least)